src/level_setup.m

```diff
@@ -0,0 +1,94 @@
+function level_collection = level_setup(directory)
+% build_level reads a directory containing level files. Level files are 
+% .txt files containing the objects to be placed in the
+% level. 
+
+% Input level_name = the name of the .txt file
+% Output level_collection = a Level array containing all levels in
+% directory.
+
+addpath(directory);
+D = dir(directory);
+ii=1;
+for nn = 1:length(D)
+    if D(nn).isdir
+        continue
+    else
+        [C,level_name] = read_level(D(nn));
+        level_collection(ii) = build_level(C,level_name);
+        ii = ii+1;
+    end
+    
+end
+
+end
+
+function [C,level_name] = read_level(D)
```

Comment:
In `read_level()`: don't return `C`, but create some more user-friendly data structure (e.g. a struct) representing the contents of the level file